spec/openapi.yaml

```diff
@@ -1697,6 +1697,51 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/404response'
+  /v1/messaging/numbers/sender_address/requests/{id}/reverify:
```

Comment:
Please update to */v1/messaging/numbers/sender_address/**addresses**/{id}/reverify*